lambda_bucket.tf

```diff
@@ -39,3 +39,15 @@ resource "aws_s3_bucket_public_access_block" "fdi_lambda" {
   ignore_public_acls      = true
   restrict_public_buckets = true
 }
+
+# Any objects placed into this bucket should be owned by the bucket
+# owner. This ensures that even if objects are Put by a different
+# account the bucket owning account retains full control over the
```

Comment:
Make this sentence a bit easier to read (IMHO):
```suggestion
# account, the bucket-owning account retains full control over the
```

Reply:
Please see [a4d50df](https://github.com/cisagov/findings-data-import-terraform/pull/38/commits/a4d50df7965c5ad4c8e83fcf7185a398fd233bf4) (I made some other tweaks at the same time).

Reply:
Looks great, thanks!